code/modules/surgery/ear_surgery.dm

```diff
@@ -0,0 +1,75 @@
+//Head surgery to fix the ears organ
+/datum/surgery/ear_surgery
+	name = "Ear surgery"
+	requires_bodypart_type = NONE
+	organ_to_manipulate = ORGAN_SLOT_EARS
+	possible_locs = list(BODY_ZONE_HEAD)
+	steps = list(
+		/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/saw,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/fix_ears,
+		/datum/surgery_step/close,
+	)
+
+//fix ears
+/datum/surgery_step/fix_ears
+	name = "fix ears (hemostat)"
+	implements = list(
+		TOOL_HEMOSTAT = 100,
+		TOOL_SCREWDRIVER = 45,
+		/obj/item/pen = 25)
+	time = 64
```

Comment:
not anything of note but was there any reason why you chose 64 time?

Reply:
Same time as eye surgery uses - I aimed to keep most of it consistent with that surgery.